### PR TITLE
update the demo example for `software.eessi.io`

### PR DESCRIPTION
**Pull request by laraPPr**
- https://github.com/EESSI/eessi-demo/pull/31

### CODE REVIEW

File: docs/using_eessi/eessi_demos.md [hunk 1]
```diff
@@ -32,30 +32,43 @@ Each of these contains a ``run.sh`` script that can be used to start a small
...
...[=====================>........] - ETA: 0s - loss: 0.0718 - accuracy: 1441/1875 [======================>.......] - ETA: 0s - loss: 0.0724 - accuracy: 1477/1875 [======================>.......] - ETA: 0s - loss: 0.0726 - accuracy: 1513/1875 [=======================>......] - ETA: 0s - loss: 0.0729 - accuracy: 1549/1875 [=======================>......] - ETA: 0s - loss: 0.0727 - accuracy: 1585/1875 [========================>.....] - ETA: 0s - loss: 0.0729 - accuracy: 1621/1875 [========================>.....] - ETA: 0s - loss: 0.0732 - accuracy: 1657/1875 [=========================>....] - ETA: 0s - loss: 0.0735 - accuracy: 1693/1875 [==========================>...] - ETA: 0s - loss: 0.0739 - accuracy: 1729/1875 [==========================>...] - ETA: 0s - loss: 0.0740 - accuracy: 1765/1875 [===========================>..] - ETA: 0s - loss: 0.0739 - accuracy: 1801/1875 [===========================>..] - ETA: 0s - loss: 0.0736 - accuracy: 1837/1875 [============================>.] - ETA: 0s - loss: 0.0736 - accuracy: 1873/1875 [============================>.] - ETA: 0s - loss: 0.0732 - accuracy: 1875/1875 [==============================] - 3s 1ms/step - loss: 0.0732 - accuracy: 0.9772
```
Comment (boegel) [inline docs/using_eessi/eessi_demos.md:68]: This is showing all "updates" per line, I think it's fine to just copy-paste the output once the command is done running?

Comment (laraPPr) [reply docs/using_eessi/eessi_demos.md:68]: This is what it looks like on the web page; It is not printing it line by line in the markdown

Screenshot (122)

Comment (boegel) [reply docs/using_eessi/eessi_demos.md:68]: OK, sure, but those hugely long lines are not very useful (and they're cut off too, see the `accurac ` part for 1st epoch).

I would use this instead (also removed those first 2-3 lines):

```
Epoch 1/5
1875/1875 [==============================] - 3s 2ms/step - loss: 0.2894 - accuracy: 0.9158
Epoch 2/5
1875/1875 [==============================] - 3s 2ms/step - loss: 0.1418 - accuracy: 0.9585
Epoch 3/5
1875/1875 [==============================] - 3s 2ms/step - loss: 0.1073 - accuracy: 0.9668
Epoch 4/5
1875/1875 [==============================] - 3s 2ms/step - loss: 0.0862 - accuracy: 0.9740
Epoch 5/5
1875/1875 [==============================] - 3s 2ms/step - loss: 0.0738 - accuracy: 0.9763
313/313 - 0s - loss: 0.0723 - accuracy: 0.9775 - 320ms/epoch - 1ms/step

real    0m21.395s
user    0m32.558s
sys     0m3.303s
```

File: docs/using_eessi/eessi_demos.md [hunk 2]
```diff
@@ -32,30 +32,40 @@ Each of these contains a ``run.sh`` script that can be used to start a small
...
+   1875/1875 [==============================] - 3s 1ms/step - loss: 0.1078 - accuracy: 0.9670
+Epoch 4/5
+   1875/1875 [==============================] - 3s 1ms/step - loss: 0.0890 - accuracy: 0.9717
+Epoch 5/5
+   1875/1875 [==============================] - 3s 1ms/step - loss: 0.0732 - accuracy: 0.9772
+313/313 - 0s - loss: 0.0679 - accuracy: 0.9790 - 391ms/epoch - 1ms/step
+
+real	1m24.645s
+user	0m16.467s
+sys	0m0.910s
```
Comment (boegel) [inline docs/using_eessi/eessi_demos.md:70]: We should use spaces here, not tabs, to make sure the times are aligned in rendered view:

```suggestion
real   1m24.645s
user   0m16.467s
sys    0m0.910s
```

**Review by boegel — CHANGES_REQUESTED** [review]

**Review by boegel — APPROVED** [review]

**Pull request merged**